job-service-acceptance-tests/src/integration-test/java/com/hpe/caf/jobservice/acceptance/JobServiceEndToEndIT.java

```diff
@@ -158,6 +163,100 @@ private void testJobCompletion(final boolean useTaskDataObject) throws Exception
         }
     }
 
+    @Test
```

Comment:
Should we include an additional test here where all of the pre-requisite job Ids have completed to make sure no rows have made it into the job_task_data and job_dependency tables?

Reply:
I have added this test in the latest commit